Makefile

```diff
@@ -246,3 +246,16 @@ golint:
 	@echo "Completed golint no recommendations !!"
 	@echo "--------------------------------"
 	@echo ""
+
+.PHONY: license-check
+license-check:
+	@echo "--> Checking license header..."
+	@licRes=$$(for file in $$(find . -type f -regex '.*\.sh\|.*\.go\|.*Docker.*\|.*\Makefile*' ! -path './vendor/*' ) ; do \
+               awk 'NR<=5' $$file | grep -Eq "(Copyright|generated|GENERATED)" || echo $$file; \
```

Comment:
what is generated|GENERATED, why we are checking fot this? @prateekpandey14

Reply:
This is for generated files, which may or may not have license , but check can ignore those files.